libs/wire-api/test/golden/Test/Wire/API/Golden/Generated/Email_user.hs

```diff
@@ -17,120 +17,81 @@
 
-testObject_Email_user_11 :: Email
+testObject_Email_user_11 :: EmailAddress
 testObject_Email_user_11 =
-  Email {emailLocal = "", emailDomain = "+zGJ\b_t/N\NUL3S\1061013M\146321\1076256z\1099407\1106566SJD"}
+  unsafeEmailAddress "some" "example"
 
-testObject_Email_user_12 :: Email
+testObject_Email_user_12 :: EmailAddress
 testObject_Email_user_12 =
-  Email
-    { emailLocal = "\EOT1[G\1014638\983349\&9\1086491:uJ\144560\FSMF\123165\985853\187923\US6|\996879\NAK\1075664",
-      emailDomain = "\96735_\1064048"
-    }
+  unsafeEmailAddress "some" "example"
 
-testObject_Email_user_13 :: Email
+testObject_Email_user_13 :: EmailAddress
 testObject_Email_user_13 =
-  Email
-    { emailLocal = "\ENQ\rt\FSA#}\RSn\176776OA\SYN\SO\1040173\t2q\DC3n\161371\185193\f+",
-      emailDomain = "]\29293\159214na[\US'h\134423\DC2\1007180\147811\1110187"
-    }
+  unsafeEmailAddress "some" "example"
 
-testObject_Email_user_14 :: Email
+testObject_Email_user_14 :: EmailAddress
 testObject_Email_user_14 =
-  Email
-    { emailLocal = "X\DLEE\DC4\1013278\1045648\1107074YMU[\n}\991766\r7\1010192\CAN\\",
-      emailDomain = "\165099\1036143M\GS!\142750T%F]"
-    }
+  unsafeEmailAddress "some" "example"
 
-testObject_Email_user_15 :: Email
-testObject_Email_user_15 = Email {emailLocal = "{C\174982\1042320eU\DC4w", emailDomain = "`\DC3>"}
+testObject_Email_user_15 :: EmailAddress
+testObject_Email_user_15 = unsafeEmailAddress "some" "example"
 
-testObject_Email_user_16 :: Email
-testObject_Email_user_16 = Email {emailLocal = "IO\EM2>\1053560+~", emailDomain = "\SO"}
+testObject_Email_user_16 :: EmailAddress
+testObject_Email_user_16 = unsafeEmailAddress "some" "example"
 
-testObject_Email_user_17 :: Email
+testObject_Email_user_17 :: EmailAddress
 testObject_Email_user_17 =
-  Email
-    { emailLocal = "\RS\1097381\SYN\SOH>\51458V7C-asF\1055340IfrYTM;\1014918\1059325*l(d",
-      emailDomain = ".53Q\1097431\&26bfw\175553\73861~\165507\131884m\GS\NAK\SO}\152927~\1051259R"
-    }
+  unsafeEmailAddress "some" "example"
 
-testObject_Email_user_18 :: Email
+testObject_Email_user_18 :: EmailAddress
 testObject_Email_user_18 =
-  Email
-    { emailLocal = "\181378|\NUL\STX3\DC2\1099608,:\ETBJuF\DLE*\15790\DC1\"\SYNkU!\989789\&8T\EM2",
-      emailDomain = "T\"q\DC1\71908}\DC3Z~\128415)"
-    }
+  unsafeEmailAddress "some" "example"
 
-testObject_Email_user_19 :: Email
+testObject_Email_user_19 :: EmailAddress
 testObject_Email_user_19 =
-  Email
-    { emailLocal = "\t\ACKN~\RSWy5'\CANq:_K\1022684\"+WM\29811S.\DC2D\DEL`\CAN",
-      emailDomain = "DG\136157A\59646E=W\1075924"
-    }
+  unsafeEmailAddress "some" "example"
 
-testObject_Email_user_20 :: Email
+testObject_Email_user_20 :: EmailAddress
 testObject_Email_user_20 =
-  Email
-    { emailLocal = "/\138192ZF\1003769\1027227",
-      emailDomain = "\187749I\1109889~FN\1016516\ENQ3PH\DC4k\1036543\131674{\1046142\ETBH\1020386\DC2IR"
-    }
+  unsafeEmailAddress "some" "example"
```

Comment:
If all the 20 test values are `unsafeEmailAddress "some" "example"`, then we as well might keep only one of them and delete the 19 JSON golden files.